shared/js/ui/models/top-blocked.es6.js

```diff
@@ -22,12 +22,13 @@ TopBlocked.prototype = window.$.extend({},
             return new Promise((resolve, reject) => {
                 this.fetch({getTopBlockedByPages: this.numCompanies})
                     .then((data) => {
-                        if (!data.totalPages || data.totalPages < 30) return resolve()
+                        if (!data.totalPages || data.totalPages < 5) return resolve()
```

Comment:
I changed this to 5 just for testing. I should change this back before we merge it.